index.js

```diff
@@ -1,7 +1,20 @@
 const readline = require('readline-sync');
+const calc = require('./calculator');
 
 console.log('Welcome to the calculator!');
 
-console.log('Please enter some input:');
-const response = readline.prompt();
-console.log('You entered: ' + response);
+console.log('Enter the operator:');
+const operator = readline.prompt();
+
+console.log(`How many numbers do you want to ${operator} ?`);
+const c = parseInt(readline.prompt());
```

Comment:
you should use a more descriptive variable name, if you came back in 3 months would you instinctively know what c is?